doc/chapters/gateways/index.rst

```diff
@@ -134,7 +134,10 @@ All the values can be substituted with the wildcard character ``*`` with the mea
 "direction" set to ``*`` will mean both incoming and outgoing, and a ``method`` set to ``*`` will
 match all method and signal names for the interface and object path specified.
 
-If a bus configuration is just an empty array it means all access to that bus will be blocked.
+If a bus configuration is just an empty array it means all access to that bus will be blocked. When
+the ``dbus-proxy`` is intercepts with D-Bus message, configuration list is filtered due to D-Bus
+message's ``direction``, ``interface``, ``path`` and ``method``. The first configuration which
+fits all arguments express that indicated dbus message is allowed.
```

Comment:
This can be reworded to something like "if a matching rule is found, the message is allowed"